tapir/shifts/views.py

```diff
@@ -82,6 +83,7 @@ def get_context_data(self, **kwargs):
         for slot in slots:
             slot.can_register = slot.user_can_attend(self.request.user)
         context["slots"] = slots
+        context["shift_is_in_the_future"] = context["shift"].start_time > timezone.now()
```

Comment:
We'll need this in quite a few places, maybe we can just make this a model method? Sort of hacky though..